cds-plugin.js

```diff
@@ -311,7 +311,9 @@ class EventBroker extends cds.MessagingService {
     } else {
       cds.app.post(webhookBasePath, _validateCertificate.bind(this))
     }
-    cds.app.post(webhookBasePath, express.json())
+
+    const limit = cds.server.body_parser?.limit ?? this.options.webhookSizeLimit ?? "1mb"
```

Comment:
```suggestion
    const limit = this.options.webhookSizeLimit ?? cds.server.body_parser?.limit ?? "1mb"
```

Personally, I'd be in favor of having the endpoint-specific configuration take precedence over the generic body parser limit. :)